content/blog/platypus-platform-pulumi-at-1000-nodes/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "The Platypus Platform: Pulumi for large-scale organizations"
+
+date: 2024-08-05
+draft: false
+social_media: "TBD"
+meta_desc: "The Platypus Platform is a comprehensive Pulumi-based internal platform for infrastructure management and secure deployments in a large-scale environment."
+meta_image: meta.png
+authors:
+    - troy-howard
+tags:
+    - platform-engineering
+    - platypus-platform
+    - devsecops
+    - architecture
+---
+
+Infrastructure management is all fun and games until you find yourself scrolling through 1000+ resources in your AWS console. Worse, when one rogue product team wants to use Azure and your data team wants to be on GCP, you're ARM wrestling in Azure and watching your economies of scale tip the wrong direction as you're copy-pasting CloudFormation templates into yet another git repo... This. Needs. To. Be. A. Platform!
+
+<!--more-->
+
+And in that moment of overwhelm, you will be sold to, nurture-emailed every week, and told all your problems will be solved by implementing an IDP (internal developer portal, as if you've never seen this acronym before). An IDP that costs a lot of money and a lot of time to implement beyond default settings. An IDP that really only solves half of your problems. Your internal team offers to build something... something that feels more like welding together random pieces of code into an abstract found-art sculpture built from junkyard refuse, already 5 years out of date. How long will this investment be useful before you have to start over?
+
+It's exhausting. If there was a good solution on the market, you wouldn't be reading this article. So let's talk about what you really need, and how Pulumi can help.
+
+## An effective internal developer platform
+
+There are quite a few listicles out there professing to authoritatively tell you the 5, or 7, or 11 essential components of an internal developer platform. Personally, I trust our customers to tell us, and here's what they have said they need:
```

Comment:
I don't know where you can drop this in, but it would be good to link to one of these pages or both if you can find it. We need better backlinking within our pages

https://www.pulumi.com/what-is/what-is-platform-engineering
https://www.pulumi.com/product/internal-developer-platforms/
https://www.pulumi.com/blog/why-switch-to-pulumi/#why-pulumi-for-internal-developer-platforms

Reply:
added